include/spdk/bdev.h

```diff
@@ -321,4 +333,6 @@ int spdk_bdev_free_io(struct spdk_bdev_io *bdev_io);
 int spdk_bdev_reset(struct spdk_bdev *bdev, enum spdk_bdev_reset_type,
 		    spdk_bdev_io_completion_cb cb, void *cb_arg);
 struct spdk_io_channel *spdk_bdev_get_io_channel(struct spdk_bdev *bdev, uint32_t priority);
+
+void spdk_bdev_set_scsi_sense(struct spdk_bdev_io *bdev_io, int sk, int asc, int ascq);
```

Comment:
I think this ought to be spdk_bdev_io_set_scsi_sense() since it operates on a bdev_io, not a bdev.

Also please leave a blank line before the #endif.